packages/components/Atoms/Global/scss/tools/_typography.scss

```diff
@@ -1,4 +1,4 @@
-@mixin rpl_typography_font($font-size: 's', $line-height: 1.2em, $font-family: 'inherit') {
+@mixin rpl_typography_font($font-size: 's', $line-height: 1.2em, $font-family: 'inherit', $include-base: true) {
```

Comment:
Hi @alan-cole Any purpose for `include-base` parameter? Looks like no place is not using it?

Reply:
This is for line 63 - 68 of this file:

> Avoid unnecessary duplication; include base styles only on first breakpoint.

When we use the breakpoint loop in rpl_typography_ruleset() for adding styles; only the font-family / line-height / font-size change, but we are also repeating all the other rules as well. This cuts down on some of that unnecessary duplication.

Reply:
@alan-cole that makes sense, thanks!